src/BuildIt.ML/BuildIt.ML.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>BuildIt.ML</id>
+    <version>$version$</version>
+    <title>BuildIt.Forms</title>
+    <authors>Built to Roam Pty Ltd</authors>
+    <owners>Built to Roam Pty Ltd</owners>
+    <licenseUrl>https://github.com/builttoroam/BuildIt/blob/master/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectUrl>https://github.com/builttoroam/BuildIt/</projectUrl>
+    <description>Simplies using machine learning in mobile applications using Xamarin.Android, Xamarin.iOS and UWP</description>
+    <copyright>Copyright � 2018</copyright>
+    <tags>BuildIt, Xamarin, Machine Learning</tags>
+  </metadata>
+  <files>
+    <file src="BuildIt.ML\bin\Release\netstandard1.0\BuildIt.ML.dll" target="lib\netstandard1.0" />
+    <file src="BuildIt.ML\bin\Release\netstandard2.0\BuildIt.ML.dll" target="lib\netstandard2.0" />
+    <file src="BuildIt.ML\bin\Release\uap10.0.16299\BuildIt.ML.dll" target="lib\uap10.0.16299" />
+
+    <file src="BuildIt.ML\bin\Release\Xamarin.iOS10\BuildIt.ML.dll" target="lib\Xamarin.iOS10" />
```

Comment:
Do you know whether source linking will work if we con't include the source in the nuget package? I'd suggest adding the source code as a fallback 
eg
`<file src="BuildIt.ML\**\*.cs" target="src\BuildIt.ML\BuildIt.ML" exclude="BuildIt.ML\obj\**\*.cs" />`

Reply:
Haven't tried so will have to check